pkg/portal/service/collaborator.go

```diff
@@ -482,6 +481,18 @@ func (s *CollaboratorService) SendInvitation(
 		}
 	}
 
+	inviteeExists, err := s.checkInviteeExistenceByEmail(ctx, invitedBy, inviteeEmail)
+	if err != nil {
+		return nil, err
+	}
+
+	if !inviteeExists {
+		err = s.createAccountForInvitee(ctx, invitedBy, inviteeEmail)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
New developers (If any) reading this part may be confused:
Why `createAccountForInvitee` is called but actually no account created?

What about:
```
// In CollaboratorService.SendInvitation
if IsAuthgearOnce {
  // ... checkInviteeExistenceByEmail
  // ... createAccountForInvitee
}

// In another file
//go:build !authgearonce
// +build !authgearonce
const IsAuthgearOnce = false

// In another file
//go:build authgearonce
// +build authgearonce
const IsAuthgearOnce = true
```

Reply:
Agree, let me make the change according to your suggestion.